packages/cli/src/evaluation.ee/test-runner/test-runner.service.ee.ts

```diff
@@ -310,9 +353,15 @@ export class TestRunnerService {
 					// Skip them, increment the failed count and continue with the next test case
 					if (!testCaseExecution) {
 						await this.testRunRepository.incrementFailed(testRun.id);
+						await this.testCaseExecutionRepository.markAsFailed(testRun.id, pastExecutionId);
```

Comment:
These should happen in a transaction. Otherwise it can end up in an inconsistent state